src/sections/Route.enum.ts

```diff
@@ -25,7 +25,6 @@ export enum QueryParamKey {
   VERSION = 'version',
   PERSISTENT_ID = 'persistentId',
   QUERY = 'q',
-  COLLECTION_ITEM_TYPES = 'types',
```

Comment:
It looks like `COLLECTION_ITEM_TYPES` was moved to `CollectionItemsQueryParams` because it is used for `CollectionSearchCriteria `- maybe `QUERY` could be moved to `CollectionItemsQueryParams` for the same reason?